backend/src/apiserver/main.go

```diff
@@ -189,6 +195,19 @@ func loadSamples(resourceManager *resource.ResourceManager) error {
 		if configErr != nil {
 			return fmt.Errorf("Failed to decompress the file %s. Error: %v", config.Name, configErr)
 		}
+		// Patch the default bucket name read from ConfigMap
+		if common.GetBoolConfigWithDefault(HasDefaultBucketEnvVar, false) {
+			defaultBucket := common.GetStringConfig(DefaultBucketNameEnvVar)
+			projectId := common.GetStringConfig(ProjectIDEnvVar)
+			patchMap := map[string]string{
+				"<your-gcs-bucket>": defaultBucket,
```

Comment:
Can/should `<your-gcs-bucket>` be a flag value? I'm wondering how else we can ensure these sentinels are kept in sync.

Reply:
Synced offline. Keep it is for now unless we have a good mechanism to sync this part with SDK.